### PR TITLE
Flatten qtconsole

### PR DESCRIPTION
**Pull request by minrk**
remove qtconsole.console subpackage

The package only has one entrypoint.

### CODE REVIEW

File: qtconsole/tests/test_console_widget.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from qtconsole.qt import QtCore, QtGui
 
 # Local imports
-from qtconsole.console.console_widget import ConsoleWidget
```
Comment (takluyver) [inline qtconsole/tests/test_console_widget.py]: You took out a bit too much here.

**Comment by takluyver** [comment]
This seems fine to me, modulo the typo that's causing a test failure.

**Comment by damianavila** [comment]
LGTM too after fixing the `No module` error...

**Comment by minrk** [comment]
Fixed.

**Pull request merged**